tests/components/ct-vue-vite/tests/render/render.spec.ts

```diff
@@ -2,6 +2,7 @@ import { test, expect } from '@playwright/experimental-ct-vue';
 import Button from '@/components/Button.vue';
 import EmptyTemplate from '@/components/EmptyTemplate.vue';
 import Component from '@/components/Component.vue';
+import { Story } from '@/components/Story';
```

Comment:
How typical is this?

Reply:
It depends on the app, but I don't it's that typical; its useful when a wrapper/story needs to be created to test components, where Playwright cannot serialise the data sent to the browser _(such as unserialisable props)_. It's possible to create multiple `.vue` files but that's quite cumbersome.

There are additional use cases where importing components from a `.ts` file would be beneficial tho, such as supporting Angular and web components _(without relying on a specific file extension)_.

Reply:
This use case is very typical for Vue and is not possible without this PR: https://github.com/microsoft/playwright/issues/31217